common/src/main/java/whocraft/tardis_refined/client/screen/waypoints/WaypointListScreen.java

```diff
@@ -115,15 +159,20 @@ public void renderBackground(GuiGraphics guiGraphics, int i, int j, float f) {
     @Override
     public void render(GuiGraphics guiGraphics, int i, int j, float f) {
 
+        int textOffset = height / 2 - 60;
+        int textScale = 40;
+
         this.renderTransparentBackground(guiGraphics);
 
         guiGraphics.blit(MONITOR_TEXTURE, leftPos, topPos, 0, 0, imageWidth, imageHeight);
 
-        if(WAYPOINTS.isEmpty()) {
-            Font font = this.font;
-            guiGraphics.drawCenteredString(font, noWaypointsLabel, width / 2, height / 2 - 10, ChatFormatting.GOLD.getColor());
+        ScreenHelper.renderWidthScaledText(Component.translatable(ModMessages.UI_MONITOR_WAYPOINTS).getString(), guiGraphics, Minecraft.getInstance().font, width / 2 - 96, textOffset, Color.LIGHT_GRAY.getRGB(), textScale * 2, 1F, false);
+
+        if (WAYPOINTS.isEmpty()) {
+            ScreenHelper.renderWidthScaledText(noWaypointsLabel.getString(), guiGraphics, Minecraft.getInstance().font, width / 2 - 96, textOffset + 15, Color.LIGHT_GRAY.getRGB(), textScale * 2, 1F, false);
```

Comment:
Move field on Line 56 to top of class file for consistency. Add access modifier i.e. public, private, protected

``Line 56:  Component noWaypointsLabel = Component.translatable(ModMessages.UI_MONITOR_NO_WAYPOINTS);``

Reply:
Addressed